src/PerfView/CommandLineArgs.cs

```diff
@@ -352,7 +352,9 @@ private void SetupCommandLine(CommandLineParser parser)
                 bool hasTpl = false;
                 foreach (var provider in onlyProviders)
                 {
-                    if (0 <= provider.IndexOf(":stack", StringComparison.OrdinalIgnoreCase))
+                    if (0 <= provider.IndexOf("@StacksEnabled=true", StringComparison.OrdinalIgnoreCase))
```

Comment:
Sadly, Contains does not have the overload that allows it to ignoring case (at least Intellisense does not give the option).   We should probably add that overload going forward.